### PR TITLE
Fem: Add smoothing filter extension to contours filter

### PR DESCRIPTION
**Pull request by marioalexis84**
Add an optional Laplacian smoothing filter to the contour filter, to get a smoother result when the generated isosurfaces are too coarse.
Below are two example images with filter disabled and filter enabled, respectively.
image
image

@FEA-eng

### CODE REVIEW

**Comment by chennes** [comment]
@FEA-eng could you take a look at this one?

**Comment by FEA-eng** [comment]
> @FEA-eng could you take a look at this one?

Sure, I’m coming back from vacation, I should be able to check it tomorrow.

**Comment by FEA-eng** [comment]
Tested, works as expected.

**Review by FEA-eng — APPROVED** [review]

**Comment by chennes** [comment]
@kadet1090 can you take a look at that conflict, with the call to get_object?

**Comment by maxwxyz** [comment]
@marioalexis84 PR has a conflict

**Comment by marioalexis84** [comment]
@maxwxyz conflict resolved

File: src/Mod/Fem/App/FemPostFilter.cpp [hunk 1]
```diff
@@ -515,6 +515,104 @@ DocumentObjectExecReturn* FemPostClipFilter::execute()
...
+    EXTENSION_ADD_PROPERTY_TYPE(FeatureAngle,
+                                (45),
+                                "Smoothing",
+                                App::Prop_None,
+                                "Angle for sharp edge identification");
+    EXTENSION_ADD_PROPERTY_TYPE(EdgeSmoothing,
+                                (false),
+                                "Smoothing",
+                                App::Prop_None,
+                                "Smooth aling sharp interior edges");
```
Comment (chennes) [inline src/Mod/Fem/App/FemPostFilter.cpp:561]:
```suggestion
                                "Smooth align sharp interior edges");
```

**Pull request merged**